ocs_ci/ocs/resources/pod.py

```diff
@@ -708,13 +710,14 @@ def get_all_pods(
     return pod_objs
 
 
-def get_ceph_tools_pod(skip_creating_pod=False, namespace=None):
+def get_ceph_tools_pod(skip_creating_pod=False, wait=False, namespace=None):
```

Comment:
Usually when we add new argument to function it's best practice to add new params at the end, then you are more safe that you don't break the current code if there is usage of positional arguments.

I did check that it's not the case in references so we are safe, just a note for next time, so we can keep it as it is as it's cherry-pick and was already merged like that to newer versions.